recipes/protobuf/all/conanfile.py

```diff
@@ -192,6 +210,8 @@ def package(self):
             tools.remove_files_by_mask(os.path.join(self.package_folder, "lib"), "libprotobuf-lite.*")
             tools.remove_files_by_mask(os.path.join(self.package_folder, "bin"), "libprotobuf-lite.*")
 
+        self._create_shim()
```

Comment:
Question, do we also need to chmod in `package_info`?
To fix accidental removal of these flags during transit?

Reply:
maybe yes, it will be more portable and safe, and harmless